easybuild/easyblocks/m/mutil.py

```diff
@@ -0,0 +1,63 @@
+##
+# Copyright 2016-2016 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# http://github.com/hpcugent/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for mutil, implemented as an easyblock
+
+@author: Ward Poelmans (Ghent University)
+"""
+import re
+import os
+
+from easybuild.easyblocks.generic.makecp import MakeCp
+from easybuild.tools.build_log import EasyBuildError
+
+
+class EB_mutil(MakeCp):
+    """Class to build mutil"""
+
+    def __init__(self, *args, **kwargs):
+        """Easyblock constructor, set correct options"""
+        super(EB_mutil, self).__init__(*args, **kwargs)
+
+        self.cfg['with_configure'] = True
```

Comment:
No, you're *not* just changing the default, you're overriding the value set in the easyconfig file, totally ignoring what may have been set in the easyconfig file...

Changing the default would be done via `extra_options`...